.github/workflows/run-build-and-acceptance-tests.yml

```diff
@@ -210,26 +222,59 @@ jobs:
         with:
           repo: t0yv0/goteststats
           tag: v0.0.7
-      - name: Ensure
+      - name: Download Pulumi Go Binaries (linux-x64)
+        if: ${{ matrix.platform == 'ubuntu-latest' }}
+        uses: actions/download-artifact@v2
+        with:
+          name: pulumi-linux-x64.tar.gz
+          path: artifacts/go
```

Comment:
Perhaps for later - but we may want to create an artifacts cleanup job

Reply:
Good point! I think it auto-cleans in 90 days, I can expose a parameter so PR verification would only keep for say 3 days.